packages/airnode-deployer/src/entrypoint.ts

```diff
@@ -18,12 +18,13 @@ export async function run(...args: unknown[]) {
   return handler.run(...args);
 }
 
-export async function processHttpRequest(...args: unknown[]) {
+// We shorten function names to allow for shorter cloud resource names
```

Comment:
I'd either copy this command above every function (preferred) or make it a top level command.